node-graph/gcore/src/ops.rs

```diff
@@ -83,6 +83,80 @@ where
 	first.pow(second)
 }
 
+// Floor
+pub struct FloorParameterNode;
+
+#[node_macro::node_fn(FloorParameterNode)]
+fn floor(input: f32) -> f32 {
+	input.floor()
+}
+
+// Ceil
+pub struct CeilParameterNode;
+
+#[node_macro::node_fn(CeilParameterNode)]
+fn ceil(input: f32) -> f32 {
+	input.ceil()
+}
+
+// Round
+pub struct RoundParameterNode;
+
+#[node_macro::node_fn(RoundParameterNode)]
+fn round(input: f32) -> f32 {
+	input.round()
+}
+
+// Absolute Value
+pub struct AbsoluteParameterNode;
+
+#[node_macro::node_fn(AbsoluteParameterNode)]
+fn abs(input: f32) -> f32 {
+	input.abs()
+}
+
+// Log
+pub struct LogParameterNode<Second> {
+	second: Second,
+}
+
+#[node_macro::node_fn(LogParameterNode)]
+fn ln<U: num_traits::float::Float>(first: U, second: U) -> U {
+	first.log(second)
+}
+
+// Natural Log
+pub struct NaturalLogParameterNode;
+
+#[node_macro::node_fn(NaturalLogParameterNode)]
+fn ln(input: f32) -> f32 {
+	input.ln()
+}
+
+// Sine
+pub struct SineParameterNode;
+
+#[node_macro::node_fn(SineParameterNode)]
+fn ln(input: f32) -> f32 {
+	input.sin()
+}
```

Comment:
The I think none of these nodes should have Parameter in their name as they don't actually take a second argument. (With AddParameter, the parameter is added to the primary input, this is not the case here)

Reply:
Sorry for that, I just copy-pasted without thinking. Should be fixed now